summary.Rmd

```diff
@@ -0,0 +1,40 @@
+Summary
+=======
+
+1. Use available tools
+    a. Use ready-made images (e.g. from Docker Hub)
+    b. Dockerfile generators (e.g. repo2docker)
+    c. Project templates (e.g. [cookie cutter](https://github.com/cookiecutter/cookiecutter))
+    d. Linters
+2. Use versioned images
+3. Format for clarity
+    a. Use indentation, line breaks and unabbreviated arguments
+    b. Clarity > brevity
+    c. Group related commands
+4. Document within the Dockerfile
+    a. Add comments explaining the steps
+    b. Use labels for metadata
+    c. Include build and run commands
+6. Specify software versions
```

Comment:
I'll rename the section to match - we don't have a "for X" in the other rules, so why here.